### PR TITLE
Neewsweek page

### PR DESCRIPTION
**Pull request by menyagah**
# In this section I did the following:
 - Created the header.
 - Added the logo.
 - Created the main section
 - Created the content section and styled everything
 - Created the footer
 - Added icons to the footer
 - Added images
 -Worked on the responsiveness of the website'
 - Corrected stickler errors
 - Validated my html file
- Changed media queries to three
- Added burger icon on the header

# Made the TSE requested changes and improvements:
- Removed the whitespace on the right most end of the main navigation by using bootstrap border 
  on the navigation instead.
- Made the top story section come first in small screens
- Made the top navigation disappear to the hamburger icon at the same breakpoint the Opinions 
  section disappears.
-  Made  the main images  fill the entire screen width for Top Story, Featured Story, In the magazine, 
   and Editor's picks for small screens.
- Validated my html and css.

# Made the TSE requested changes :
- Used the bootstrap grid system and flexbox on my header. 

# Made the TSE requested changes
- Worked on centering the webpage using container class instead of the container-fluid class
- Worked on my html by removing the excwssive use of divs

# Made the TSE requested changes
- Worked on the overflowing navbar
- Worked on the mobile view for the debate section

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,665 @@
...
+  display: flex;
+  align-items: center;
+  grid-column: 1 / 2;
+  grid-row: 1 / 3;
+}
+
+.second {
+  grid-column: 2 / 3;
+  grid-row: 1 / 3;
+}
```
Comment (trekab) [inline css/style.css]: This is a great opportunity to rely on bootstrap classes. 
For example [Grid layout](https://getbootstrap.com/docs/4.3/layout/overview/) and [Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options)).([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/using_bootstrap/bootstrap_classes_requirement.png) from [Pathwrigth](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540308/)). This applies to all your custom media queries at the bottom.

**Review by trekab — CHANGES_REQUESTED** [review]
# Changes Required!
Hi @menyagah, well done on this project so far! However, there are several changes that you need to make to complete this project. These include;

### Mandatory:
- [ ] There is too much white space to the rightmost end of the main navigation.
- [ ] The top navigation should disappear to the hamburger icon at the same breakpoint the `Opinions` section disappears from the top to the bottom. Your clone appears to have different breaking points for this.
- [ ] `TOP STORY` should come first on small screens.
- [ ] For small screens, the main images should always fill the entire screen width for `Top Story`, `Featured Story`, `In the magazine`, and `Editor's picks`.
- [ ] As indicated in the comment within the code, you need to rely on bootstrap as much as possible.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review_ :wink: 
__Treasure Kabareebe__

File: css/style.css (hunk 1 above)
Comment (menyagah) [reply css/style.css]: Thank you for the timely review! I am working on the changes.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,683 @@
...
+
+@media only screen and (max-width: 765px) {
+  .top-story-1 {
+    display: inline;
+  }
+
+  .top-story {
+    display: none;
+  }
+}
```
Comment (trekab) [inline css/style.css]: You should also rely on bootstrap classes for [responsive break-points](https://getbootstrap.com/docs/4.3/layout/overview/#responsive-breakpoints).

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,683 @@
...
+.top {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.next-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (trekab) [inline css/style.css:403]: This is another opportunity to use bootstrap classes. ([Flex](https://getbootstrap.com/docs/4.3/utilities/flex/))

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,683 @@
...
+  height: 30px;
+}
+
+.under-button-logo {
+  display: flex;
+  background-color: white;
+  justify-content: center;
+  align-items: center;
```
Comment (trekab) [inline css/style.css:368]: This is another opportunity to use bootstrap classes. ([Flex](https://getbootstrap.com/docs/4.3/utilities/flex/))

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,683 @@
...
+  padding-bottom: 2px;
+}
+
+.my-card {
+  background-color: #f4f5f6;
+}
+
+.top-story::-webkit-scrollbar {
+  display: none;
+}
```
Comment (trekab) [inline css/style.css:86]: This is another place you could rely on bootstrap classes. ([Display](https://getbootstrap.com/docs/4.3/utilities/display/))

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,683 @@
...
+  font-weight: 400;
+}
+
+.top-story {
+  height: 1810px;
+  overflow-y: scroll;
+}
+
+.card-text {
+  display: flex;
```
Comment (trekab) [inline css/style.css:76]: This is another opportunity to use bootstrap classes. ([Flex](https://getbootstrap.com/docs/4.3/utilities/flex/))

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,683 @@
...
+  text-decoration: none;
+}
+
+.logo-1 {
+  width: auto;
+}
+
+.mobile {
+  display: none;
+}
```
Comment (trekab) [inline css/style.css]: This is another place you could rely on bootstrap classes. ([Display](https://getbootstrap.com/docs/4.3/utilities/display/))

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,683 @@
...
+.icon-header,
+.icon-header-1 {
+  padding-bottom: 3px;
+  color: white;
+  font-size: 900;
+}
+
+.icon-burger {
+  display: none;
+}
```
Comment (trekab) [inline css/style.css:45]: This is another place you could rely on bootstrap classes. ([Display](https://getbootstrap.com/docs/4.3/utilities/display/))

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,683 @@
...
+  grid-column: 3 / 4;
+  grid-row: 1 / 3;
+}
+
+.third-cont {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding-right: 30px;
+}
```
Comment (trekab) [inline css/style.css]: This is a perfect opportunity to use bootstrap classes. ([Flex](https://getbootstrap.com/docs/4.3/utilities/flex/))

**Review by trekab — CHANGES_REQUESTED** [review]
# Changes Required!
Hi @menyagah! I have noticed that you have implemented some of the earlier code review requirements. Well done! I know how overwhelming this project can be and the hard work you are putting in is commendable. However, It appears that you are missing the core requirement and main skill that this project is supposed to help you acquire. There are several opportunities in your CSS where you have a chance to rely on bootstrap classes. You should use Bootstrap CSS classes as much as possible for styling ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/using_bootstrap/bootstrap_classes_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540308/)).
I have tried to highlight several areas in your CSS file where you can rely on bootstrap CSS classes in the comments in the code.
__Please make the necessary changes and submit another code review request__
_Feel free to reach me on slack for any inquiries about this review_ :wink:
__Treasure Kabareebe__

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @menyagah  :wave:
-
You have done a splendid job! :tada: I laud the effort that has been put in this project. Keep it up.
We, however, do have some few changes to carry out before we proceed (see review comment).
##
- [ ] The cloned page is currently not centred as expected. I did notice some usage of `container-fluid` class to style our content. I'd advise you wrap the entire content in a `container` class to have their contents well centred as with the source. See the difference below. 👇  

|[Source](https://www.newsweek.com/)|[Clone](https://raw.githack.com/menyagah/Newsweekclone/neewsweek-page/index.html)|
|:---:|:---:|
|source|clone|
 
- [ ] Take note also, of the usage of [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) elements. They should be "appropriately sized and positioned" (as described in Odin project description) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/semantic_html_requirement.png) from Pathwright).

- [ ]  Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by santiago-dev96** [comment]
# Pull Request under review by @santiago-rodrig (TSE)

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required 🚫

You are doing an outstanding job with this project, but there are some remaining issues.

Screen Shot 2020-05-22 at 9 41 19 AM

The last element of the navigation bar (the search element) is not displaying properly, fix that; other thing that needs to be fixed is the "The Debate" section.

Screen Shot 2020-05-22 at 9 42 12 AM

You can always look at [the original page](https://www.newsweek.com) to compare.

After implementing the required changes, please submit another CRR.

You can find me in slack as **srodrig**. :wave:

**Comment by menyagah** [comment]
Thank you for the review! I really appreciate it. Let me work on it and
resubmit it.

**Review by rammazzoti2000 — APPROVED** [review]
### Hello @menyagah you've come a long way, congrats :tada: :rocket: 
Your project is ready for approval, although there would be some more issues to address, but given that you've implements all the changes required by the previous TSEs, I will aprove this project, with the follow given conditions:
- [ ] There is a contrast difference in the navbar between the logo background color and the background color of the navbar, you can see it for yourself in the screenshit provided. To fix this problem you should use a pickcolor tool to match the logo and the navbar background and to avoid this contrast difference
image
  - [ ] The bottom logo with the social links on small screens should be aligned on the left, yours is centered
image
###
Cheers and Happy coding! :clap: 

Code review by Alexandru Bangau
### <em>Kindly fix the REQUIRED changes before merging, thank you :pray: 
(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>
### :heavy_check_mark:  Status: Approved

**Comment by menyagah** [comment]
I highly appreciate your review. I will definitely fix the issues pointed
out. Once again thank you for your approval.

**Pull request merged**